pkg/amplitude/amplitude.go

```diff
@@ -235,6 +236,7 @@ func identify() {
 	timezone, _ := time.Now().In(time.Local).Zone()
 
 	builder := ampli.Identify.Builder().
+		DdevEnvironment(environment.GetDDEVEnvironment()).
```

Comment:
Can't we just use Environment here? The Ddev prefix looks superfluous to me.

Reply:
Talking about the Amplitude property, I don't care much about the func name here.